### ISSUE TITLE
Major Enhancement - Responsiveness General

### ISSUE THREAD

**Issue by GoodGameRuler** [GoodGameRuler/unitrack_v3#46]
TLDR: Need for some amount of resting on different resolution and planning of different sizing

Factors to Consider
- [ ] True Screen Size
- [ ] Exposed Browser Viewport Size
- [ ] Any browser-viewport css
- [ ] Device type, touch...

What the task entails
- Going and testing different screen, devices, and browsers
- Testing consistency and practicality of css sizing and positioning of (most pertinent to the `width` attribute. and checking need for min and max width)

A good place to start is the `pageContentsWrapper` and its contents. For example the `detailsForm` as part of the `degreePlanner` page is not responsive. 

Consider margins, gaps, and spacing for all these factors.

**Comment by GoodGameRuler** [comment]
See issue #27 for some responsiveness concerns.
